proc.c

```diff
@@ -439,13 +439,13 @@ scheduler(void)
         switch (pr->state)
         {
           case RUNNING:
-            pr->rutime++;
+            pr->rutime += (delta_ticks / QUANTA);
```

Comment:
I don't understand this logic. A) why aren't we modifying the existing IRQ_TIMER handler, B) why does it matter if the proc is running, and C) what's up with the yield?